app/controllers/adminUsers/employees_controller.rb

```diff
@@ -23,6 +23,27 @@ def destroy
       redirect_to admin_users_employees_path, notice: 'Employee was successfully destroyed.'
     end
 
+    def add_kudos
+      render :add_kudos
+    end
+
+    def increase_number_of_available_kudos
+      number_of_kudos = params[:number_of_kudos].to_i
+      if number_of_kudos.between?(1, 20)
+        Employee.transaction do
+          Employee.all.each do |e|
```

Comment:
Please do not use one letter variable names ;)
```suggestion
          Employee.all.each do |employee|
```